check_postgres.pl

```diff
@@ -5789,9 +5800,17 @@ sub find_new_version {
             ## Postgres is slightly different
             if ($program eq 'Postgres') {
                 $cmajor = {};
-                while ($info =~ /<title>(\d+)\.(\d+)\.(\d+)/g) {
-                    $found = 1;
-                    $cmajor->{"$1.$2"} = $3;
+                if ($version >= 10) {
+                    while ($info =~ /<title>(\d+)\.(\d+)/g) {
+                        $found = 1;
+                        $cmajor->{"$1.$2"} = $3;
```

Comment:
I suspect the problem is in this line. The regular expression being matched to doesn't have a `$3`. I'm guessing here, but would
```
                        $cmajor->{"$1"} = $2;
```
be appropriate instead?

Reply:
hi @esabol - based on local testing that appears to be correct; thanks.  Will fixup and merge.

Reply:
I wonder if there should be `[^\.]` after the `(\d+)\.(\d+)` in the regular expression on line 5804. Otherwise, it will match the x.y.z versions as well, and I think the intent is to only match the x.y versions?